docs/first_pr.md

```diff
@@ -0,0 +1,61 @@
+# Getting started with EvalAI Issues:
+
+To contribute for CloudCV in EvalAI is a facile task. Please follow these steps in order to get started :-)
+
+**Step 1 : Fork**
+
+1. Fork the EvalAI repository from [this](https://github.com/Cloud-CV/EvalAI) link.
+
+**Step 2 : Picking up a issue:**
+
+1. Pick up a suitable issue that you find will be facile for you to fix. Moreover, you can also 
+take the issues based on their labels. Almost every issue is marked with a label according to its difficulty.
+
+2. After selecting any issue, mention any mentor’s name there and ask him to assign it to you.
+
+3. Once it gets assigned , create a branch from your fork’s updated master branch using the command :-
+`git checkout -b branch_name`
+**Note:** To know more about the `checkout` command command, please refer to [this](https://git-scm.com/docs/git-checkout) link.
+
+4. Start working on the issue.
+
+**Step 3: Committing Your Changes:**
+
+1. After making the required feature as asked in the issue, you need to add your files to local git repository.
+
+2. If you want to create any migration file then use this command to create the migration file.
+`python manage.py makemigrations app_name --name=suitable_name --settings=settings.dev`
+
+3. To add your files , these commands can be handy:
+
+- To add only modified files `git add -u`
+- To add a new file `git add file_path_from_local_git_repository`
+- To add all files `git add .`
+
+4. Once you have added your files, you need to commit your changes. Always **create a very meaningful commit message related to the feature that you have added.**
+
+**Step 4: Creating a Pull Request:**
+
+**Note:** To know more about **Pull Request**, please refer to [this](https://help.github.com/articles/about-pull-requests/) link.
+
+1. Before creating a Pull Request, you need to first rebase your branch with the upstream master.
```

Comment:
I think we can remove this part `To know more about` from the whole doc and instead add links in the part itself. Here stackoverflow link can be included in first point itself, like

Before creating a Pull Request, you need to first rebase your branch with the [upstream](http://stackoverflow.com/questions/9257533/what-is-the-difference-between-origin-and-upstream-on-github) master